templates/istio.yaml

```diff
@@ -35,7 +35,7 @@ spec:
   chartReference:
     chart: istio
     repo: https://mesosphere.github.io/charts/staging
-    version: 1.3.3
+    version: 1.3.5
```

Comment:
@GoelDeepak take note, I point kubeaddons-configs to the latest istio chart in mesosphere/charts.